docs/_future/concepts.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Slack developer beta
+lang: en
+slug: concepts
+order: 0
+layout: future
+permalink: /future/concepts
+---
+
+# Slack developer beta <span class="label-beta">BETA</span>
```

Comment:
@hello-ashleyintech not sure I correctly understand the question, if this is regarding opting to use `/future/concepts#<a concept>` over using `/future/<a concept>` this was to reduce the number of network requests and provide a crisp experience once  `/future/concepts` is loaded, it also reflect the `basic concept` pattern

let me know if I correctly understood the question

Reply:
Yes, that sounds good! That makes sense to me. It differs from the JS docs, so that might be an opportunity to evaluate the JS docs and see how we can consolidate it into a similar page concept depending on the volume of information there